src/com/google/enterprise/adaptor/GsaFeedFileSender.java

```diff
@@ -257,12 +259,12 @@ public void sendFull(String datasource, String xmlString,
    * Groupsource name is limited to [a-zA-Z_][a-zA-Z0-9_-]*.
    */
   void sendGroups(String groupsource, String xmlString,
-      boolean useCompression) throws IOException {
+      boolean useCompression, boolean incremental) throws IOException {
```

Comment:
Most of the other methods like these have the xmlString as the last param.  At some point, someone added useCompression at the end, then imysak added incremental after that.